### PR TITLE
Color test

### PR DESCRIPTION
**Pull request by jayesh-parmar**

### CODE REVIEW

**Comment by jayesh-parmar** [comment]
Screenshot (36)

File: tests/Feature/ColorControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    $this->post(route('color.update', $color->id), [
+        'id' => $color->id,
+        'name' => 'update red'
+    ])
+        ->assertRedirect(route('color.index'));
+
+    $this->assertDatabaseHas('colors', [
+        'name' => 'update red',
+    ]);
```
Comment (rajan-parmar) [inline tests/Feature/ColorControllerTest.php]:
```suggestion
    $this->assertDatabaseHas('colors', [
    'id' => $color->id,
        'name' => 'update red',
    ]);
```

**Review by rajan-parmar — CHANGES_REQUESTED** [review]

**Review by rajan-parmar — APPROVED** [review]

File: tests/Feature/ColorControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?php
```
Comment (gauravmak) [inline tests/Feature/ColorControllerTest.php:1]: @rajan-parmar Do we plan to cover any more functionalities?

**Review by gauravmak — CHANGES_REQUESTED** [review]

File: tests/Feature/ColorControllerTest.php (hunk 2 above)
Comment (rajan-parmar) [reply tests/Feature/ColorControllerTest.php:1]: > @rajan-parmar Do we plan to cover any more functionalities?

based on functionality it's ok for now. for testing others are pending.

**Pull request merged**